runtime/CMakeLists.txt

```diff
@@ -150,6 +150,13 @@ else()
 	add_custom_target(j9vm_nlsgen)
 endif()
 
+# Generate jvmti.h
+add_custom_command(OUTPUT "${CMAKE_CURRENT_SOURCE_DIR}/include/jvmti.h"
+   	COMMAND m4 -D "JAVA_SPEC_VERSION=${JAVA_SPEC_VERSION}" "${CMAKE_CURRENT_SOURCE_DIR}/include/jvmti.h.m4" > "${CMAKE_CURRENT_SOURCE_DIR}/include/jvmti.h"
```

Comment:
The working directory need not be repeated in the command; it could be:
>    	COMMAND m4 -D "JAVA_SPEC_VERSION=${JAVA_SPEC_VERSION}" "jvmti.h.m4" > "jvmti.h"